tsl/src/remote/dist_copy.c

```diff
@@ -1333,6 +1335,8 @@ remote_copy_process_and_send_data(RemoteCopyContext *context)
 	Hypertable *ht = context->ht;
 	const int n = context->batch_row_count;
 	Assert(n <= MAX_BATCH_ROWS);
+	static int32 chunk_id = INVALID_CHUNK_ID;
```

Comment:
Why do we need this variables to be static as these are being reset when the function returns?

Reply:
It needs to be static for the loop to cache the previous chunk_id. Assuming that nearby inserts will go to the same chunk, we this ways, avoid additional DN scans. @pdipesh02

Reply:
Hmm, yes, good point @pdipesh02 will remove the statics!